tutorials/colors/colorbar_only.py

```diff
@@ -0,0 +1,108 @@
+"""
+=============================
+Customized Colorbars Tutorial
+=============================
+
+This tutorial shows how to build colorbars without an attached plot.
+
+"""
+
+###############################################################################
+# Customized Colorbars
+# ====================
+#
+# ColorbarBase derives from ScalarMappable and puts a colorbar in a specified
+# axes, so it has everything needed for a standalone colorbar. It can be used
+# as is to make a colorbar for a given colormap and does not need a mappable
+# object like an image. In this tutorial we will explore what can be done with
+# standalone colorbar.
+#
+# We will start by making a figure of desired size and adding thress axes.
+
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+
+fig, (ax1, ax2, ax3) = plt.subplots(nrows=3)
+
+###############################################################################
+# Basic continuous colorbar
+# -------------------------
+#
+# Set the colormap and norm to correspond to the data for which the colorbar
+# will be used. Then create the colorbar by calling `ColorbarBase` and
+# specify axis, colormap, norm and orientation as parameters. Here we create
+# a basic continuous colorbar with ticks and labels. There are many more kwargs
+# which can be used to further modify the colorbar.
```

Comment:
In these kinds of sentences I like to say something like "for more examples, see <link-to-colorbar-examples-if-they-exist>". Something worth thinking of inserting where reasonable!